storage-node/packages/cli/bin/dev.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable no-console */
+
+'use strict'
+
+const debug = require('debug')('joystream:storage-cli:dev')
+const assert = require('assert')
```

Comment:
I was using it to assert later that filledMap.size was equal to one, then replaced it with a check and throwing, so forgotr to remove that import. Honestly I had my linter turned off while coding because of a huge number of warnings I could hardly see the code :)

Perhaps its a good idea now that I have working code to turn the linter back on and fix all the issues?

Reply:
it is actually used in `assert_file()`